Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node
```

```suggestion
FROM node:18
```
краще вказувати версію